projects/fusion-ui/components/table/v3/components/table-basic/table-basic.component.html

```diff
@@ -9,13 +9,21 @@
         [isRowSelected]="tableService.isRowSelected(row)"
         [isExpanded]="isExpanded(rowIndex)"
         [isInnerRow]="false"
+        [class.is-row-in-request]="tableService.isRowInRequest(row)"
```

Comment:
Added it as CSS class for input row. Because in case of row-span rows we  need to set this class to an amount rows more than one